packages/client/src/components/set/index.js

```diff
@@ -1,43 +1,48 @@
-import { useParams } from "react-router-dom";
-import { useEffect, useState } from 'react';
-import { Container, Row, Table, Spinner } from 'react-bootstrap';
-import { setEV } from '../../utils/calculator';
-import { getSetInfo } from '../../api';
+import {useParams} from "react-router-dom";
+import {useEffect, useState} from 'react';
+import {Container, Row, Spinner} from 'react-bootstrap';
+import {setEV} from '../../utils/calculator';
+import {getSetInfo} from '../../api';
 import DataTable from "./table";
 import CalcAccordion from "./accordion";
+import SetHeader from "./header";
 
-function Set({ mode }) {
-    let { setCode } = useParams();
+
+function Set({mode}) {
+    let {setCode} = useParams();
     const [packValue, setPackValue] = useState(null);
     const [boxValue, setBoxValue] = useState(null);
     const [filteredPackValue, setFilteredPackValue] = useState(null);
     const [filteredBoxValue, setFilteredBoxValue] = useState(null);
     const [totalSetData, setTotalSetData] = useState(null);
-    const [setName, setSetName] = useState(null);
+    const [setData, setSetData] = useState(null);
     const [errors, setErrors] = useState(false);
 
 
     const getSetData = async () => {
         try {
-            const { packValue, boxValue, filteredPackValue, filteredBoxValue, totalSetData } = await setEV(setCode);
+            const {packValue, boxValue, filteredPackValue, filteredBoxValue, totalSetData} = await setEV(setCode);
             if (packValue || boxValue || filteredPackValue || filteredBoxValue || totalSetData) {
                 setPackValue(packValue);
                 setBoxValue(boxValue);
                 setFilteredPackValue(filteredPackValue);
                 setFilteredBoxValue(filteredBoxValue);
                 setTotalSetData(totalSetData);
             }
-            let { name } = await getSetInfo(setCode);
-            if (name) {
-                setSetName(name);
+            let data = await getSetInfo(setCode);
+            if (data) {
+                console.log(data);
+                setSetData(data);
             }
```

Comment:
looks like my IDE did some formatting, but...

I just converted setName -> setData so we could use more info on the page if we wanted to (icon, card count, etc...)